clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -267,6 +267,42 @@ class ClioWebClient(
     ).flatMap(unmarshal[Json])
   }
 
+  def getMetadataForKey[CI <: ClioIndex](clioIndex: CI)(
```

Comment:
This version of the method shouldn't be in here... rebase gone wrong?

Reply:
ugh probably

Reply:
This is still screwy... the method is already defined on line 214 above